docs/about.md

```diff
@@ -55,16 +55,51 @@ Contributor [2021 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Gokul Rajiv](https://github.com/gok99)
+### [Zhou Jiahao](https://github.com/zhoukerrr)
 **Role**: Contributor [2022 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Zhou Jiahao](https://github.com/zhoukerrr)
+### [Charisma Kausar](https://www.github.com/ckcherry23)
+**Role**: Contributor [2024 January - ]<br/>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [David Gareth Ong](https://www.github.com/vvidday)
+**Role**: Contributor [2024 January - ]<br/>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [Gokul Rajiv](https://github.com/gok99)
 **Role**: Contributor [2022 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
+### [Marcus Tang Xin Kye](https://www.github.com/MarcusTXK)
+**Role**: Contributor [2024 January - ]<br/>
+
+<!-- ------------------------------------------------------------------------------------------------------ -->
+
+### [Alvis Ng](https://www.github.com/supermii2)
+**Role**: Developer [2024 January - ]<br/>
```

Comment:
Use 'Contributor' instead of 'Developer'
```suggestion
**Role**: Contributor [2024 January - ]<br/>
```

Reply:
Got it! Will update and commit!